Joining Main with Varan (trial1)

Lindaalnsour opened the pull request:
-Enhancement of the main & remove all the apps from it.
-Integrate the first two modules correctly into main & ensure they run well in the terminal as long within Varan API. 
-Correct the url for module 2 & return the response valid in json format.
- Sanitising the user input in the API (remove the white space and lower case the input)

Lindaalnsour committed:
-Enhancement of the main & remove all the apps from it.
-Integrate the first two modules correctly into main & ensure they run well in the terminal as long within Varan API.
-Correct the url for module 2 & return the response valid in json format.
- Sanitising the user input in the API (remove the white space and lower case the input)

tophie97 reviewed (approved):
Most of the changes get the stamp of approval from me! If possible I think adding which SPDI the result is sourced from would be helpful. Also, it seems that some of the print functions are a bit further down than they should be, however, I will double-check this once the code is merged. 

Some of the module 3 print code is commented instead of implemented but I recall this being intentional before so we can confirm that shortly. Otherwise, I am pleased that it seems capable of getting GRCh37 and 38 as requested by our customer. 

Lastly, I note that some requirements have moved location but are still present and consistent with what was there before, this is good. I look forward to us testing the functionality soon!

tophie97 commented:
After review, it seems that team-1 branch has some requirements we need to add before finishing the merge.

Lindaalnsour committed:
Merge branch 'main' of github.com:gl186/SprintFinish into team-1

# Conflicts:
#	Modules/Main.py
#	Modules/module2_variantvalidator.py
#	Modules/module3_VV_LOVD_code_only.py
#	Modules/module5_SPDI.py
#	requirements.txt

Lindaalnsour committed:
- Sanitising the user input in the API (remove the white space).
- resolve the conflict.
- debug the API to know where was the problem that didn't allow us to get out the results.
